Assets/MirageXR/ContentTypes/3DModel/Model.cs

```diff
@@ -113,11 +113,18 @@ private async Task<bool> LoadGltf(ToggleObject content)
         {
             content.option = ZipUtilities.CheckFileForIllegalCharacters(content.option);
             var loadPath = Path.Combine(RootObject.Instance.activityManager.ActivityPath, content.option, GLTF_NAME);
+
```

Comment:
The PR is based on the fwild/1928-model-animation-fix branch which has not yet been approved. update your PR.

Reply:
now merged - do I still need to rebase this?